nft-v2/Makefile

```diff
@@ -0,0 +1,40 @@
+.PHONY: init candid build local test format lint clean
+
+OWNER_CANISTER_PRINCIPAL=$(shell cat canister-owner-test-principal.txt)
+
+init:
+	npm i
+	cargo check
+
+candid:
+	cargo run > nft-v2.did
+	didc bind -t ts nft-v2.did  > test/factory/idl.d.ts
+	echo "// @ts-nocheck" > test/factory/idl.ts
+	didc bind -t js nft-v2.did  >> test/factory/idl.ts
+
+build: candid
+	dfx stop
+	dfx start --clean --background
```

Comment:
We should move these 2 lines into a separate make script, just in case someone wants to use a replica that's already running

Reply:
cool, it should be achieved with this 7c62de86968159c165071c888846a852004f97f6